pkgs/racket-doc/scribblings/reference/syntax-util.scrbl

```diff
@@ -141,13 +141,15 @@ the parameter has a non-false value. This is done automatically by
 forms like @racket[with-disappeared-uses].
 }
 
-@defform[(with-disappeared-uses stx-expr)
+@defform[(with-disappeared-uses body-expr ... stx-expr)
```

Comment:
There *is* a difference, though: the `body-expr`s can evaluate to anything, but the `stx-expr` must evaluate to a syntax object. I can clarify the prose a little bit more to make that clear, though (the last bit should probably read “Adds the caught identifiers to the `'disappeared-uses` syntax property of the syntax object produced by `stx-expr`.”).